entries/SWC-134.md

```diff
@@ -0,0 +1,15 @@
+# Title
+Avoid Using Block Number as a Proxy for Time
+
+## Relationships
+[CWE-829: Inclusion of Functionality from Untrusted Control Sphere](https://cwe.mitre.org/data/definitions/829.html)
+
+## Description
+Generally, the block time on Ethereum is about 14 seconds. Given this knowledge it's possible to predict the time delta between blocks. However, block times are not constant and are subject to change for a variety of reasons, e.g. fork reorganisations and the difficulty bomb. Due to variable block times, `block.number` should not be relied on for precise calculations of time. 
+
+## Remediation
+Precise calculations of time should not be done using `block.number`. Instead, developers may make use of oracles.
```

Comment:
I think using `block.timestamp` is acceptable for some use cases and use of an oracle whenever you deal with some sort time period period might be overkill?